apps/dokploy/components/dashboard/swarm/monitoring-card.tsx

```diff
@@ -28,13 +30,68 @@ export default function SwarmMonitorCard({ serverId }: Props) {
 
 	if (isLoading) {
 		return (
-			<div className="w-full max-w-7xl mx-auto">
-				<div className="mb-6 border min-h-[55vh] rounded-lg h-full">
-					<div className="flex items-center justify-center h-full text-muted-foreground">
-						<Loader2 className="h-6 w-6 animate-spin" />
-					</div>
+    <div className="min-h-screen">
+			<div className="w-full max-w-7xl mx-auto space-y-6 py-4">
+				<div className="space-y-4">
+					<Skeleton className="h-8 w-1/3" />
+					<Skeleton className="h-5 w-2/5" />
 				</div>
+
+				<div className="grid gap-6 md:grid-cols-3">
+					{Array.from({ length: 3 }).map((_, i) => (
+						<div key={i} className="p-4 border rounded-md">
+							<div className="flex flex-row items-center justify-between mb-4">
+								<Skeleton className="h-4 w-1/4" />
+								<Skeleton className="h-8 w-8 rounded-md" />
+							</div>
+							<Skeleton className="h-6 w-1/2" />
+						</div>
+					))}
+				</div>
+
+        <Card className="w-full bg-background">
```

Comment:
Indendation inconsistency here :/

Maybe you should fix it ?

Reply:
Piggy-backing off this comment, I'd happy to create a PR with Prettier with current conventions + Tailwind plugin to sort Tailwind classes to avoid mistakes like this. Ping me and I'll set some time for that.